spec/generators/pageflow/assets/assets_generator_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'support/generators'
+require 'fileutils'
+
+# Generators are not automatically loaded by Rails
+require 'generators/pageflow/assets/assets_generator'
+
+module Pageflow
+  module Generators
+    describe AssetsGenerator, type: :generator do
+      setup_default_destination
+
+      before do
+        FileUtils.mkdir_p "#{destination_root}/app/assets/javascripts"
+        FileUtils.mkdir_p "#{destination_root}/app/assets/stylesheets"
+        FileUtils.mkdir_p "#{destination_root}/config"
+
+        FileUtils.touch "#{destination_root}/app/assets/javascripts/active_admin.js.coffee"
```

Comment:
@tf your thoughts about how deep these tests should go?

In this example I've just touched the file that is required by this generator. This is cool with me, as it allows us to run this test in the first place. I would be reluctant to actually re-create the entire contents of these files. It seems like that is very brittle.

So I propose that as far as required files by these generators, I just `touch` them, but never recreate the contents for them.